tracex_project/extraction/logic/orchestrator.py

```diff
@@ -96,8 +91,7 @@ def initialize_modules(self):
             self.configuration.modules["time_extraction"](),
             self.configuration.modules["event_type_classification"](),
             self.configuration.modules["location_extraction"](),
-            # This module should be activated only if the user wants to analyze the metrics
-            # self.configuration.modules["metrics_analyzer"](),
+            self.configuration.modules["metrics_analyzer"](),
             # Only activate this module with a test comparison patient journey as ground truth
             # self.configuration.modules["event_log_comparator"](),
```

Comment:
still using old naming

Reply:
Do you mean the "event_log_comparator" vs trace_comparator?

Reply:
yess

Reply:
If so, it's fixed now